lxd/container_lxc.go

```diff
@@ -3228,6 +3231,44 @@ func (c *containerLXC) Export(w io.Writer) error {
 			return err
 		}
 	} else {
+		if properties != nil {
+			// Parse the metadata
+			content, err := ioutil.ReadFile(fnam)
+			if err != nil {
+				tw.Close()
+				return err
+			}
+
+			metadata := new(imageMetadata)
+			err = yaml.Unmarshal(content, &metadata)
```

Comment:
Nope, they're meant to be yaml. The images on images.linuxcontainers.org are json just because the script I use to generate them doesn't use external python dependencies and there is no native yaml support.

Since yaml is a superset of json, this works fine.